### PR TITLE
chore: cleanup get utxosfunction

### PR DESCRIPTION
**Pull request by andredevjs**
Cleans up a bit the logic, to make it easier to move to nostr-core and have better unit tests later.

### CODE REVIEW

File: src/utils/crypto.js [hunk 1]
```diff
@@ -140,3 +143,49 @@ export function tweakSigner(signer) {
...
+    const vout = view.getInt32(0, 1);
+    return [txid, vout];
+};
+
+export function sortUtxos(utxos) {
+    const sortedData = utxos.sort((a, b) => b.status.block_time - a.status.block_time);
+    return sortedData.map((utxo) => ({ ...utxo, key: `${utxo.txid}:${utxo.vout}` }));
+}
+
+export const getInscriptionsForAddress = async (address) => {
```
Comment (habibitcoin) [inline src/utils/crypto.js:170]: nit: better in inscriptions.js?

File: src/utils/crypto.js [hunk 2]
```diff
@@ -140,3 +143,49 @@ export function tweakSigner(signer) {
...
+    const response = await axios.get(`${TURBO_API}/wallet/${address}/inscriptions`);
+    return response.data;
+};
+
+export const getOutpointFromCache = async (inscriptionId) => {
+    const key = `${LocalStorageKeys.INSCRIPTIONS_OUTPOINT}:${inscriptionId}`;
```
Comment (habibitcoin) [inline src/utils/crypto.js:176]: nit: better in inscriptions.js?

Comment (habibitcoin) [reply src/utils/crypto.js:176]: think it keeps things cleaner, and quite like this segregation of responsibilities within files

**Review by habibitcoin — APPROVED** [review]
Very nice! 1 nit. QA passed

File: src/utils/crypto.js (hunk 2 above)
Comment (andredevjs) [reply src/utils/crypto.js:176]: agree

**Pull request merged**